src/microbe_stage/PlayerHoverInfo.cs

```diff
@@ -6,13 +6,16 @@
 /// </summary>
 public class PlayerHoverInfo : Node
 {
+    private readonly Dictionary<Compound, float> currentHoveredCompounds = new Dictionary<Compound, float>();
+    private readonly Dictionary<Compound, float> compoundDelayTimer = new Dictionary<Compound, float>();
     private MicrobeCamera camera;
     private CompoundCloudSystem cloudSystem;
+    private Vector3 lastCursorWorldPos = Vector3.Inf;
```

Comment:
```suggestion

    private Vector3? lastCursorWorldPos;
```

can use a nullable type here to indicate the invalid value